demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -72,7 +72,7 @@ def __init__(
             print_ignored_files=False, skip_conf_json=True, validate_id_set=False, file_path=None,
             validate_all=False, is_external_repo=False, skip_pack_rn_validation=False, print_ignored_errors=False,
             silence_init_prints=False, no_docker_checks=False, skip_dependencies=False, id_set_path=None, staged=False,
-            skip_id_set_creation=False
+            skip_id_set_creation=True
```

Comment:
This is a logic change - why do we do this?
if id_set is not created pack dependencies is not validated which is something you do want locally if possible as far as I know.

Also if we do keep it we need to update `__main__` as the flag description should change to `--create-id-set` as the default now is not to do this.

Reply:
By default we should not create it, but use the id_set we get from the buvket. we can always override and force creation with the given arg.